README.md

```diff
@@ -11,7 +11,7 @@ make generate
 ## Testing
 
 ```shell script
-make test
+make e2eTest
```

Comment:
I would leave this `make test` since the test should include all the tests (including e2e). But if you want to just run e2e you can make another command `make e2eTest`

Reply:
Looks like that got changed automatically in the readme when I did the rename, but not actually in the makefile anyways haha. Changed this back :).